lib/Backend/LowerMDShared.cpp

```diff
@@ -1835,6 +1838,17 @@ void LowererMD::LegalizeOpnds(IR::Instr *const instr, const LegalForms dstForms,
     {
         LegalizeSrc<verify>(instr, instr->GetSrc2(), NormalizeForms(src2Forms));
     }
+
+    //hasMemOpnd |= instr->GetSrc2()->IsMemoryOpnd();
```

Comment:
What's the plan with this chunk?

Reply:
I tried to delete this 3 times and all 3 times it crashed the program...I thought my 4th try deleted it but I must not have saved...

---
In reply to: [204248337](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r204248337) [](ancestors = 204248337)